webapp/src/components/testing/Testing.vue

```diff
@@ -129,6 +129,17 @@
           As such, please ensure your dataset of choice was not used during the training phase of your model.
         </template>
       </IconCard>
+      <!-- Language model prompting is currently unavailable   -->
+      <div 
+        v-if="currentTask.id === 'llm_task' && validationStore.isOnlyPrediction"
```

Comment:
more resilient to changes

```suggestion
        v-if="currentTask.trainingInformation.dataType === 'text' && validationStore.isOnlyPrediction"
```